src/comm_manager.jl

```diff
@@ -97,7 +97,7 @@ end
 
 # handlers for incoming comm_* messages
 
-function comm_open(sock, msg)
+function comm_open(sock, msg, kernel)
```

Comment:
```suggestion
function comm_open(sock, kernel, msg)
```
needed to match the signature from the event loop (I think?)